### DISCUSSION TITLE
Multiple sites with one database

### DISCUSSION THREAD

**Discussion by samrathchadha** [prisma/prisma#11621] [Q&A] [answered]
Is it possible to share migrations across projects and what would this look like in production?

**Comment by pantharshit00** [comment, marked as answer]
Can you please elaborate on what do you mean by sharing "migrations"? If you use the same schema in all your projects, then yes, you can share the migrations as well as they will restore the database to the same state. But you are also saying the title that you are using only one database so you will required to be execute those migrations only once.

**Reply by samrathchadha** [reply]
I meant that I didn't want prisma/migrations folder to be outdated in either of the projects. Basically, some sort of system to sync the migrations

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
